src/ivar.cpp

```diff
@@ -17,7 +17,7 @@
 #include "suffix_tree.h"
 #include "trim_primer_quality.h"
 
-const std::string VERSION = "1.3.1";
+const std::string VERSION = "1.3.2";
```

Comment:
Should this be `1.3.3` instead of `1.3.2`?

Reply:
That commit is now just on top of the last tagged for `1.3.2` —ideally it would be the one tagged for `1.3.2`. Since `1.3.1` was the version shown during all the years of development before releasing `1.3.2`, it seems that it is right to keep `1.3.2` now for the sake of coherency. (Whether developers might decide to change version control to show `1.3.3dev` or similar is a different discussion I guess.)